fastapi_backend/tests/main/test_main.py

```diff
@@ -74,7 +74,7 @@ class TestPasswordValidation:
             ),
         ],
     )
-    @pytest.mark.asyncio(loop_scope="session")
+    @pytest.mark.asyncio(loop_scope="function")
```

Comment:
Can you define a function loop_scope for all tests? I guess this would be the default / most commonly used?

Reply:
I'll merge this and remove on the next pr, with the proper test configuration.